core/ui_mixins/quick_panel.py

```diff
@@ -168,7 +162,7 @@ def __init__(self, items, on_done, **kwargs):
         self.on_done = on_done
         for option in ['flags', 'selected_index', 'on_highlight',
                        'limit', 'next_message', ]:
-            if option in kwargs:
+            if option in kwargs and kwargs[option]:
```

Comment:
right, it should be `kwargs[option] is not None`. If the nullness is not checked, it may overwrite the default methods, e.g `on_highlight`.